scripts/functions/installer

```diff
@@ -1101,11 +1101,12 @@ setup_user_profile_rc()
       touch "$profile_file"
       printf "%b" "
 ###############################################################################
-# At $(date --iso-8601=s), the RVM installer added this block (and also created
-# this file if it did not already exist).
+# At $(date --iso-8601=s), this code block (and also this file, if it did not
+# already exist) was created by:
+# $__rvm_version
```

Comment:
Did you try ${rvm_version}?

If that does not work we might need to wait until my PR #4043 is reviewed by @mpapis and merged.